doc/api/n-api.md

```diff
@@ -369,7 +369,7 @@ Native addons may need to allocate global state of which they make use during
 their entire life cycle such that the state must be unique to each instance of
 the addon.
 
-To this env, N-API provides a way to allocate data such that its life cycle is
+To this end, N-API provides a way to allocate data such that its life cycle is
```

Comment:
I'd believe this is talking about `napi_set_instance_data`, and I'm not sure this is a typo, maybe @gabrielschulhof can clarify on this.

Reply:
We discussed in latest N-API team meeting and it is a typo, and this PR shouldland.

Reply:
@legendecas it is a typo – though I have to admit, it makes sense even if we don't consider it a typo. I guess it's a pun – unintentionally.

Reply:
Thanks everyone!